docs/Tracking model classes list.md

```diff
@@ -0,0 +1,14 @@
+# Available tracking events model classes
+* VSDKAddToCart
```

Comment:
Why does this class have a different URL path than the others?

For instance, at https://ios.developers.velocidi.com/:
 - https://ios.developers.velocidi.com/Classes.html#/c:objc(cs)VSDKAddToCart
 - https://ios.developers.velocidi.com/Classes/VSDKPageView.html

Building the docs from this branch I get:
 - docs/docs_page/Other%20Classes.html#/c:objc(cs)VSDKAddToCart
 - docs/docs_page/Classes/VSDKPageView.html

Reply:
Good catch! I have no idea why that occurs 😅 I'll look into it, but it might be an error with _jazzy_

Reply:
Any luck?

Reply:
As far as I can tell, the structure of the source code and comments is really similar, there should be no difference in the way it is generated. Must be a bug 🤷‍♂

Reply:
I also think it is related to jazzy. We can skip it for now and investigate this later.